modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-fonts.adoc

```diff
@@ -0,0 +1,65 @@
+[[fonts]]
+== Fonts
+
+During document writing, the possibility of using many different fonts can be very important to users.
+
+Using the appropriate font can change the appearance of the document and emphasize its style.
+
+{pluginname} Converter allows link:https://exportpdf.converter.tiny.cloud/docs#section/Web-Fonts[Web Fonts^] to be used, which provided the integrator with the ability to use standard operating system fonts or use custom fonts without the need to import them using CSS.
+
+Below is a list of the basic fonts included in the image:
+
+[source]
+----
+OpenSans-Bold.ttf
+OpenSans-BoldItalic.ttf
+OpenSans-ExtraBold.ttf
+OpenSans-ExtraBoldItalic.ttf
+OpenSans-Italic.ttf
+OpenSans-Light.ttf
+OpenSans-LightItalic.ttf
+OpenSans-Regular.ttf
+OpenSans-Semibold.ttf
+OpenSans-SemiboldItalic.ttf
+----
+
+However, additional fonts can be added to {pluginname} Converter in two ways:
+
+* Use Unix-like PDF-Converter image `{dockerimageexporttopdf}` and mount fonts directory to it.
+** See xref:individual-export-to-pdf-on-premises.adoc#add-custom-fonts-to-pdf-converter[Add custom fonts to PDF Converter] section.
+* Use Windows PDF-Converter image `{dockerimageexporttopdf}` and mount to it fonts directory from the Windows operating system on which the container is running.
+** See Use Windows fonts in PDF Converter section.
+
+[NOTE]
+The fonts inside the mounted volume will be installed on the docker image operating system. Only the `.ttf` and `.otf` font formats are supported. If other font formats are used, these will need to be converted to the supported format prior or use fonts such as link:https://exportpdf.converter.tiny.cloud/docs#section/Web-Fonts[Web Fonts^].
+
+[TIP]
+Ensure that the converted fonts can be installed and used on your local machine first, before installing them on the docker container.
+
+[[add-custom-fonts-to-pdf-converter]]
+=== Add custom fonts to PDF Converter
+
+If custom fonts are being used in PDF files, use the `pdf-converter` Docker image and mount the directory with the custom fonts for the PDF Converter application running on a machine with a Unix-like system (this includes Docker on Windows with a WSL backend).
```

Comment:
@ShiridiGandham please confirm this is accurate.
```suggestion
If custom fonts are being used in PDF files, use the `pdf-converter-tiny` Docker image and mount the directory with the custom fonts for the PDF Converter application running on a machine with a Unix-like system (this includes Docker on Windows with a WSL backend).
```

Reply:
Confirmed.